client/agent-opentracing/src/main/resources/apmrules/org/apache/apm-apache-http-client.btm

```diff
@@ -35,3 +35,14 @@ IF hasSpan()
 DO
   finishSpan();
 ENDRULE
+
+RULE org.apache.http.client(4) Apache HttpClient Producer Sync Exception
+CLASS ^org.apache.http.impl.client.CloseableHttpClient
+METHOD doExecute
+HELPER org.hawkular.apm.agent.opentracing.OpenTracingManager
+AT EXCEPTION EXIT
+IF hasSpan()
+DO
+  getSpan().setTag("error","true").setTag("http.status_code",String.valueOf(503));
```

Comment:
Sure will remove.
Error tag is set on this same line :)
Yes will be providing a test.